src/php/Command/CommandConfig.php

```diff
@@ -4,29 +4,21 @@
 
 namespace Phel\Command;
 
-use Phel\AbstractPhelConfig;
-use RuntimeException;
+use Gacela\Framework\Config;
+use Phel\PhelAbstractConfig;
 
-final class CommandConfig extends AbstractPhelConfig implements CommandConfigInterface
+final class CommandConfig extends PhelAbstractConfig
 {
+    /**
+     * @return string[]
+     */
     public function getDefaultTestDirectories(): array
```

Comment:
Why is this named `default`? `getTestDirectories` would be a better name I think.